package.json

```diff
@@ -27,12 +27,15 @@
     "@docusaurus/preset-classic": "^3.0.0-beta.0",
     "@docusaurus/utils": "^3.0.0-beta.0",
     "@docusaurus/utils-common": "^3.0.0-beta.0",
+    "@flashbots/mev-share-client": "^0.7.10",
```

Comment:
I think this can go in devDependencies -- we're only using an interface from it, right?

Reply:
good catch. type info is removed in compilation so it won't be used in runtime. will change